src/main/java/WordsAnalyser.java

```diff
@@ -6,79 +6,86 @@
 import java.util.List;
 
 public class WordsAnalyser {
-    private static final int CHAR_TO_INT_HELPER = 97;
-    private static final int LATIN_ALPHABET_LENGTH = 26;
-    private final List<String>[][] buckets;
-    private String originalWord;
-    private String longestWord;
-    private String secondLongestWord;
-
-    public WordsAnalyser() {
-        buckets = new ArrayList[LATIN_ALPHABET_LENGTH][LATIN_ALPHABET_LENGTH];
-        for (int i = 0; i < buckets.length; i++) {
-            for (int j = 0; j < buckets[i].length; j++) {
-                buckets[i][j] = new ArrayList<>();
-            }
-        }
-        longestWord = secondLongestWord = "";
-    }
-
-    public List<String> examineUrl(URL file) {
+    public static List<String> getConcatenatedWordsFromUrl(URL file) {
         List<String> wordList = new ArrayList<>();
-        try (BufferedReader reader = new BufferedReader
-                (new InputStreamReader(file.openStream()))) {
+        ResultData resultData = new ResultData();
+        try (BufferedReader reader
+                     = new BufferedReader(new InputStreamReader(file.openStream()))) {
             String word;
             while ((word = reader.readLine()) != null) {
                 if (!word.isEmpty()) {
                     wordList.add(word);
-                    buckets[findRow(word)][findCol(word)].add(word);
+                    resultData.sortWord(word);
                 }
             }
         } catch (IOException e) {
             System.out.println("There is no such file!");
         }
-        return getAllConcatenatedWords(wordList);
+        return resultData.getAllConcatenatedWords(wordList);
     }
 
-    public String getLongestWord() {
-        return longestWord;
+    public static String getLongestWord() {
```

Comment:
```suggestion
    public String getLongestWord() {
```
can we remove static from WordsAnalyser's methods?

Reply:
made those methods non-static)